Examples/Movies/android/app/src/main/java/com/facebook/react/movies/MoviesActivity.java

```diff
@@ -35,7 +36,7 @@ protected void onCreate(Bundle savedInstanceState) {
 
     mReactInstanceManager = ReactInstanceManager.builder()
         .setApplication(getApplication())
-        .setBundleAssetName("MoviesApp.android.bundle")
+        .setJSBundleLoader(JSBundleLoader.createAssetLoader("MoviesApp.android.bundle"))
```

Comment:
On iOS you can re-use the same bridge with a different sourceURL. "reload" is a bit of a misnomer here since you're loading a different URL, but you could imagine something like: 
```
bridge.sourceURL = "http://localhost:8081/bundle.js?version=1"
... later in the program ...
bridge.sourceURL = "http://localhost:8081/bundle.js?version=2"
bridge.reload();
```